src/core/Form/SearchInput/SearchInput.md

```diff
@@ -10,6 +10,7 @@ Examples:
 - [Debounce](./#/Components/SearchInput?id=debounce)
 - [Full width](./#/Components/SearchInput?id=full-width)
 - [Hidden label](./#/Components/SearchInput?id=hidden-label)
+- [Search suggestions](./#/Components/SearchInput?id=search-suggestions)
```

Comment:
At the top of the file, it still reads "The component is intended to be used as part of a traditional page search in the header of a website." This is probably not the case anymore?